python/dgl/data/tu.py

```diff
@@ -276,7 +276,7 @@ class TUDataset(DGLBuiltinDataset):
     Notes
     -----
     Graphs may have node labels, node attributes, edge labels, and edge attributes,
-    varing from different dataset. This class does not perform additional process.
+    varing from different dataset. 
```

Comment:
How about the following appended to the existing notes:

> Labels are mapped to :math:`\{0,...,n-1\}` where :math:`n` is the number of labels (some datasets have raw labels :math:`\{-1, 1\}` which will be mapped to :math:`\{0, 1\}`). In previous versions, the minimum label was added so that :math:`\{-1, 1\}` was mapped to :math:`\{0, 2\}`.

Reply:
That sounds good to me